lib/account/account-service.js

```diff
@@ -128,6 +181,43 @@ module.exports = function (config, secureHash, storage) {
 
 
     /**
+     * Makes a challenge key from hashes and directory names.
+     * Errors when config settings are not in place because we
+     * do not want a directory to have undefined in the key.
```

Comment:
Agreed - we don't want "undefined" in the key, though config validation should be a separate module.